app_personal_blog/src/ui/form/Login.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react'; // хук для хранения состояния (когда компоненет первый раз ренедрится происходит инициализация)
```

Comment:
It's better to move form to feature components. It's not clean UI element; some business logic exists here